ds-css/package.json

```diff
@@ -33,15 +34,39 @@
     "bulma": "^0.7.5",
     "clean-webpack-plugin": "^3.0.0",
     "css-loader": "^3.2.0",
+    "eslint": "^6.5.1",
     "eslint-loader": "^3.0.2",
     "extract-text-webpack-plugin": "^4.0.0-beta.0",
     "html-webpack-plugin": "^3.2.0",
+    "husky": "^3.0.9",
     "mini-css-extract-plugin": "^0.8.0",
     "node-sass": "^4.12.0",
     "optimize-css-assets-webpack-plugin": "^5.0.3",
+    "resolve-url-loader": "^3.1.0",
     "sass-loader": "^8.0.0",
+    "stylelint": "^11.0.0",
+    "stylelint-config-recommended": "^3.0.0",
+    "stylelint-scss": "^3.11.1",
+    "stylelint-webpack-plugin": "^1.0.2",
     "webpack": "^4.41.0",
     "webpack-cli": "^3.3.9",
     "webpack-dev-server": "^3.8.1"
+  },
+
+"scripts": {
+  "clean": "rimraf dist",
+  "lint:scss": "> tests/linting/stylelint-errors.log && stylelint \"src/styles/custom/**/*.scss\" --fix --output-file tests/linting/stylelint-errors.log",
```

Comment:
What about the scss in the root style directory?  Why not have the stylelint-errors.log in the root directory since it is just a temporary file and add to the gitignore list?

Reply:
﻿﻿ok, i removed the tests/ folder and placed the errors log in the root.

Reply:
and i added to the .gitignore file